### PR TITLE
IGVF-1662-add-protocols-sample-objects

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/whole_organism.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * Add calculated property `lower_bound_age_in_hours`.
 * Update `dbxrefs` regex to allow ENCODE sample accessions.
 * Add `originated_from`.
+* Add `protocols`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/whole_organism.md:10]: new change on top

**Review by gabdank — CHANGES_REQUESTED** [review]
see pr comments

File: src/igvfd/schemas/changelogs/primary_cell.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 * Add calculated property `lower_bound_age_in_hours`.
 * Update `dbxrefs` regex to allow ENCODE sample accessions.
 * Add `originated_from`.
+* Add `protocols`.
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/schemas/changelogs/primary_cell.md:10]: done

File: src/igvfd/schemas/changelogs/multiplexed_sample.md [hunk 3]
```diff
@@ -30,6 +30,10 @@
 * Add `construct_library_sets`.
 * Expand `collections` enum list to include `ClinGen`, `GREGoR`, `IGVF_catalog_beta_v0.1`, and `MaveDB`.
 
+### Minor change since schema version 4
+
+* Add `protocols`.
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/schemas/changelogs/multiplexed_sample.md:35]: done

File: src/igvfd/schemas/changelogs/technical_sample.md [hunk 4]
```diff
@@ -3,6 +3,7 @@
 ### Minor changes since schema version 12
 
 * Update `dbxrefs` regex to allow ENCODE sample accessions.
+* Add `protocols`.
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/schemas/changelogs/technical_sample.md:6]: done

File: src/igvfd/schemas/changelogs/tissue.md [hunk 5]
```diff
@@ -7,6 +7,7 @@
 * Add calculated property `lower_bound_age_in_hours`.
 * Update `dbxrefs` regex to allow ENCODE sample accessions.
 * Add `originated_from`.
+* Add `protocols`.
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/schemas/changelogs/tissue.md:10]: done

**Review by gabdank — DISMISSED** [review]

**Review by gabdank — APPROVED** [review]

**Pull request merged**